extension/js/common/composer.ts

```diff
@@ -412,22 +412,19 @@ export class Composer {
     await this.composerContacts.setEmailsPreview(this.getRecipients());
     await this.renderComposeTable();
     if (this.canReadEmails) {
-      const determined = await this.app.emailProviderDetermineReplyMsgHeaderVariables();
-      if (determined) {
-        this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
-        this.composerSendBtn.additionalMsgHeaders['In-Reply-To'] = determined.headers['In-Reply-To'];
-        this.composerSendBtn.additionalMsgHeaders.References = determined.headers.References;
-        if (!this.urlParams.draftId) { // if there is a draft, don't attempt to pull quoted content. It's assumed to be already present in the draft
-          (async () => { // not awaited because can take a long time & blocks rendering
-            const footer = this.getFooter();
-            await this.composerQuote.addTripleDotQuoteExpandBtn(determined.lastMsgId, method, footer);
-            if (this.composerQuote.messageToReplyOrForward && this.composerQuote.messageToReplyOrForward.isSigned) {
+      this.urlParams.subject = `${(method === 'reply' ? 'Re' : 'Fwd')}: ${this.urlParams.subject}`;
+      if (!this.urlParams.draftId) { // if there is a draft, don't attempt to pull quoted content. It's assumed to be already present in the draft
```

Comment:
Currently, when sending from a draft, what will happen?

Reply:
Everything is good, the message was sent in the same thread